ui/packages/ui/src/app/pages/createConnector/connectorSteps/SelectConnectorTypeComponent.tsx

```diff
@@ -61,11 +93,14 @@ export const SelectConnectorComponent: React.FunctionComponent<ISelectConnectorC
         >
           {() => (
             <Flex>
-            {connectorTypes.map((cType, index) => (
-              <FlexItem key={index} className={"select-connector-component_cardItem"}>
-                <Card>
+            {getSortedConnectorTypes(connectorTypes).map((cType, index) => (
+              <FlexItem key={index} className={"select-connector-type-component_cardItem"}>
+                <Card id={cType.className} 
```

Comment:
WDYT of changing the text colour for the disabled card to indicate the disable state. something like below.
![image](https://user-images.githubusercontent.com/8264372/90366700-3e606b80-e085-11ea-8e6c-97632523e928.png)
![image](https://user-images.githubusercontent.com/8264372/90366736-4e784b00-e085-11ea-8e80-12052371a4fb.png)